site/About-the-Foundation/security.md

```diff
@@ -0,0 +1,46 @@
+# IVI Foundation Security
+
+The IVI Foundation is committed to providing secure software to its users. We take security seriously and appreciate the efforts of users and security researchers to help us improve the security of our software. We encourage the community to participate in our responsible disclosure process.
+
+## Responsible disclosure
+
+If you believe you have found a security issue in our software, we encourage you to notify us immediately. We welcome working with you to resolve the issue promptly.
+
+To report security issues, contact us at:
+
+[security@ivifoundation.org](mailto:security@ivifoundation.org)
+
+Please include the following information in your report:
+
+1. A clear description of the issue and steps to reproduce it.
+2. The version of the software in use.
+3. Operating system and version details.
+4. Any extra information that may help us understand the issue.
+5. Your name and contact information.
+6. Whether you want credit for the discovery.
+
+## Our commitment
+
+From the moment we receive a report, we will:
+
+1. Confirm the receipt of the report; typically within a business day.
+2. Assign a person to investigate the issue.
+3. Provide an estimated time frame for the resolution of the issue; typically within 30 days.
+4. Keep you informed with the progress of the investigation.
+5. Work with you to coordinate public disclosure in a way that minimizes risk to our users.
```

Comment:
This is appropriately vague, but I would still avoid hinting that we will ever do much of a public disclosure of a security problem.  (I think usually, the software update just has something like "this release corrects defects and security issues".

I am not sure how that impacts a security researcher that is looking for credit, but I think it is the norm.

How about: "Work with you to coordinate releasing updated components that minimizes risk to our users."